helm/airflow/values.yaml

```diff
@@ -1,724 +1,710 @@
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
-  nodeSelector: {}
-  affinity: {}
-  tolerations: []
-  ##
-  ## Directory in which to mount secrets on webserver nodes.
-  secretsDir: /var/airflow/secrets
-  ##
-  ## Secrets which will be mounted as a file at `secretsDir/<secret name>`.
-  secrets: []
-
-##
-## Workers configuration
-workers:
-  enabled: true
-  ##
-  ## Number of workers pod to launch
-  replicas: 1
-  ##
-  ## Gracefull termination period
-  terminationPeriod: 30
-  ##
-  ## Custom resource configuration
-  resources: {}
-    # limits:
-    #   cpu: "1"
-    #   memory: "2G"
-    # requests:
-    #   cpu: "0.5"
-    #   memory: "512Mi"
-  ##
-  ## Labels for the Worker statefulSet
-  labels: {}
-  ##
-  ## Annotations for the Worker statefulSet
-  annotations: {}
-  ##
-  ## Annotations for the Worker pods
-  podAnnotations: {}
-    ## Example:
-    ## iam.amazonaws.com/role: airflow-Role
-  ##
-  ## Celery worker configuration
-  celery:
-    ##
-    ## number of parallel celery tasks per worker
-    instances: 1
-    ##
-    ## Gracefull termination of workers
-    ## Wait for the worker to finish the current task within the gracefull period
-    gracefullTermination: false
-
-  ##
-  ## Directory in which to mount secrets on worker nodes.
-  secretsDir: /var/airflow/secrets
-  ##
-  ## Secrets which will be mounted as a file at `secretsDir/<secret name>`.
-  secrets: []
-
-  ## Support Node, affinity and tolerations for worker pod assignment
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
-  nodeSelector: {}
-  affinity: {}
-  tolerations: []
-
-##
-## Ingress configuration
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+# Default values for airflow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# User and group of airflow user
+uid: 1002490000
+gid: 1002490000
```

Comment:
Those need to be changed based on the namespace. We may want multiple values.yml files